sound/soc/sof/sof-priv.h

```diff
@@ -639,4 +640,9 @@ int sof_stream_pcm_close(struct snd_sof_dev *sdev,
 			 struct snd_pcm_substream *substream);
 
 int sof_machine_check(struct snd_sof_dev *sdev);
+
+static inline int snd_sof_get_abi_major(struct snd_sof_dev *sdev)
```

Comment:
Can you rename it to `sof_get_abi_major()`, it is not sound related

Reply:
I suppose this new helper is going to be used in other places than in the workaround for the PM?

Reply:
> Can you rename it to sof_get_abi_major(), it is not sound related

Done

> I suppose this new helper is going to be used in other places than in the workaround for the PM?

Yes, we will use it in many places where we need to know the ABI major version.